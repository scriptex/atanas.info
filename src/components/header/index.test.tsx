import * as React from 'react';
import { shallow } from 'enzyme';

import { Header } from '.';

describe('Header component', () => {
	it('Should render the Header component', () => {
		const wrapper = shallow(<Header />);

		expect(wrapper).toMatchSnapshot();
	});
});
