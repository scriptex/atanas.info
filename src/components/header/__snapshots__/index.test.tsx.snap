// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Should render the Header component 1`] = `
<header
  className="c-header"
>
  <a
    className="c-logo"
    href="#hello"
    title="Back to top"
  >
    <Component
      className="c-svg-logo"
      name="svg-logo"
    />
  </a>
  <button
    aria-label="Toggle menu"
    className="c-nav__toggle"
    name="menu-toggle"
    onClick={[Function]}
  >
    <span />
    <span />
    <span />
  </button>
  <nav
    className="c-nav"
  >
    <ul>
      <li
        key="0"
      >
        <a
          href="#hello"
          onClick={[Function]}
          title="Back to top"
        >
          Home
        </a>
      </li>
      <li
        key="1"
      >
        <a
          href="#about"
          onClick={[Function]}
          title="About me"
        >
          About
        </a>
      </li>
      <li
        key="2"
      >
        <a
          href="#timeline"
          onClick={[Function]}
          title="Life events"
        >
          Timeline
        </a>
      </li>
      <li
        key="3"
      >
        <a
          href="#skills"
          onClick={[Function]}
          title="My skills"
        >
          Skills
        </a>
      </li>
      <li
        key="4"
      >
        <a
          href="#portfolio"
          onClick={[Function]}
          title="My portfolio"
        >
          Portfolio
        </a>
      </li>
      <li
        key="5"
      >
        <a
          href="#stats"
          onClick={[Function]}
          title="My stats"
        >
          Stats
        </a>
      </li>
      <li
        key="6"
      >
        <a
          href="#slides"
          onClick={[Function]}
          title="My presentations"
        >
          Slides
        </a>
      </li>
      <li
        key="7"
      >
        <a
          href="#videos"
          onClick={[Function]}
          title="My video presentations"
        >
          Videos
        </a>
      </li>
      <li
        key="8"
      >
        <a
          href="#articles"
          onClick={[Function]}
          title="My published articles"
        >
          Articles
        </a>
      </li>
      <li
        key="9"
      >
        <a
          href="#certificates"
          onClick={[Function]}
          title="My certificates"
        >
          Certificates
        </a>
      </li>
      <li
        key="10"
      >
        <a
          href="#social"
          onClick={[Function]}
          title="Social widgets"
        >
          Social
        </a>
      </li>
      <li
        key="11"
      >
        <a
          href="./resume.pdf"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
          title="My CV"
        >
          Resume (pdf)
        </a>
      </li>
      <li
        key="12"
      >
        <a
          href="https://scriptexbg.wordpress.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
          title="My blog"
        >
          Blog
        </a>
      </li>
      <li
        key="13"
      >
        <a
          href="projects/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
          title="My open source projects"
        >
          Open source
        </a>
      </li>
      <li
        key="14"
      >
        <a
          href="music/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
          title="My music"
        >
          Music
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
